include/cutlass/fast_math.h

```diff
@@ -365,7 +365,7 @@ struct FastDivmod {
   FastDivmod(int divisor): divisor(divisor) {
 
     if (divisor != 1) {
-      unsigned int p = 31 + find_log2(divisor);
+      unsigned int p = 31 + unsigned(find_log2(divisor));
```

Comment:
This relies on "`int` + `unsigned`" implicitly converting to `unsigned`, and also mentions "`unsigned`" twice.  Would you instead consider converting the entire expression to `unsigned int` and using `auto` on the left-hand side, to avoid repetition?

```suggestion
      auto p = static_cast<unsigned int>(31 + find_log2(divisor));
```